sklearnex/linear_model/linear.py

```diff
@@ -163,8 +163,11 @@ def _onedal_fit_supported(self, method_name, *data):
         n_samples = _num_samples(X)
         n_features = _num_features(X, fallback_1d=True)
 
-        # Check if equations are well defined
+        # Note: support for some variants was either introduced later on,
+        # or had bugs in some uncommon cases in older versions.

```

Comment:
Please be more specific, i.e. "Multiple output regression was introduced in oneDAL 2025.1", etc.
Maybe also makes sense to move this comment closer to the checks.